.github/workflows/nodeci.yml

```diff
@@ -28,9 +28,15 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - run: npm install
     - run: npm run build --if-present
+    - run: '(test -f dist-server/db.js && echo db exists) || echo DB file not exists'
```

Comment:
@gutorc92, can you explain to me the lifecycle of this `db.js` and why woudn't it exis here?

Reply:
It was just a test. The test was working on my machine but they does not work on github actions. I fixed it already and remove this line.